sdk/mediaservices/arm-mediaservices/README.md

```diff
@@ -24,25 +24,83 @@ npm install @azure/arm-mediaservices
 npm install @azure/ms-rest-nodeauth@"^3.0.0"
 ```
 
+##### Create a .env file to use with the sample code
+
+- Create a .env file in your project root to use with the "dotenv" module in the sample code block. 
+- Values for this file can be obtained in the Azure Media Services API Access page in the portal. 
+
+```
+# copy the content of this file to a file named ".env". It should be stored at the root of the repo.
+# The values can be obtained from the API Access page for your Media Services account in the portal.
+AZURE_CLIENT_ID=""
+AZURE_CLIENT_SECRET= ""
+AZURE_TENANT_ID= ""
+
+# Change this to match your AAD Tenant domain name. 
+AAD_TENANT_DOMAIN = "microsoft.onmicrosoft.com"
+
+# Set this to your Media Services Account name, resource group it is contained in, and location
+AZURE_MEDIA_ACCOUNT_NAME = ""
+AZURE_LOCATION= ""
+AZURE_RESOURCE_GROUP= ""
+
+# Set this to your Azure Subscription ID
+AZURE_SUBSCRIPTION_ID= ""
+
+# You must change these if you are using Gov Cloud, China, or other non standard cloud regions
+AZURE_ARM_AUDIENCE= "https://management.core.windows.net"
+AZURE_ARM_ENDPOINT="https://management.azure.com"
+```
+
 ##### Sample code
 
 ```typescript
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
```

Comment:
We typically don't add the copy right when the code snippets are inlined in the readme file.

Reply:
Understood. Will remove.